File: Web/Models/IssueOption.cs
using System.ComponentModel.DataAnnotations;

namespace VotingModelLibrary.Models
{
    public class IssueOption
    {
        [Key]
        public int IssueOptionId { get; set; }

        public string IssueOptionTitle { get; set; }
        public string IssueOptionInfo { get; set; }

        public int BallotIssueId { get; set; }
        public BallotIssue BallotIssue { get; set; }
    }
}

File: Spa/ClientApp/src/app/models/IssueOption.ts
export class IssueOption{
  issueOptionId: number;
  issueOptionTitle: string;
  issueOptionInfo: string;
  ballotIssueId: number;

  constructor(
    issueOptionId: number,
    issueOptionTitle: string,
    issueOptionInfo: string,
    ballotIssueId: number
  ) {
    this.issueOptionId = issueOptionId;
    this.issueOptionTitle = issueOptionTitle;
    this.issueOptionInfo = issueOptionInfo;
    this.ballotIssueId = ballotIssueId
  }
}
